src/main/java/it/pagopa/transactions/controllers/v2/TransactionsController.java

```diff
@@ -70,11 +70,12 @@ public Mono<ResponseEntity<NewTransactionResponseDto>> newTransaction(
         return newTransactionRequest
                 .flatMap(ntr -> {
                     log.info(
-                            "newTransaction rptIDs {} ",
+                            "Create new Transaction for rptId: {}. ClientId: {} ",
```

Comment:
what about using [] braces around log values also for this and above logs too?

Reply:
great!

Reply:
done